runtime-light/stdlib/rpc/rpc-api.h

```diff
@@ -0,0 +1,113 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <concepts>
+#include <cstdint>
+#include <memory>
+
+#include "runtime-core/runtime-core.h"
+#include "runtime-light/coroutine/task.h"
+#include "runtime-light/stdlib/rpc/rpc-extra-info.h"
+#include "runtime-light/stdlib/rpc/rpc-tl-error.h"
+#include "runtime-light/stdlib/rpc/rpc-tl-function.h"
+#include "runtime-light/stdlib/rpc/rpc-tl-kphp-request.h"
+
+constexpr int64_t RPC_VALID_QUERY_ID_RANGE_START = 0;
+constexpr int64_t RPC_INVALID_QUERY_ID = -1;
+constexpr int64_t RPC_IGNORED_ANSWER_QUERY_ID = -2;
+
+namespace rpc_impl_ {
+
+struct RpcQueryInfo {
+  int64_t id{RPC_INVALID_QUERY_ID};
+  size_t request_size{0};
+  double timestamp{0.0};
+};
+
+task_t<RpcQueryInfo> typed_rpc_tl_query_one_impl(const string &actor, const RpcRequest &rpc_request, double timeout, bool collect_responses_extra_info,
+                                                 bool ignore_answer) noexcept;
+
+task_t<class_instance<C$VK$TL$RpcResponse>> typed_rpc_tl_query_result_one_impl(int64_t query_id, const RpcErrorFactory &error_factory) noexcept;
+
+} // namespace rpc_impl_
+
+// === Rpc Store ==================================================================================
+
+bool f$store_int(int64_t v) noexcept;
```

Comment:
Why functions.txt does not include below functions?

Reply:
We don't need these function right now. That's why I didn't add them into `functions.txt` at the moment